package-overrides/npm/aws-sdk@2.4.0.json

```diff
@@ -0,0 +1,8 @@
+{
+  "main": "dist/aws-sdk",
+  "format": "global",
+  "exports": "AWS",
+  "map": {
+    "process": "@empty"
+  }
```

Comment:
If you add `"jspmNodeConversion": false` then the map config becomes unnecessary here.